projects/plugins/jetpack/extensions/blocks/premium-content/_inc/subscription-service/class-token-subscription-service.php

```diff
@@ -106,6 +107,57 @@ public function visitor_can_view_content( $valid_plan_ids ) {
 		return $this->validate_subscriptions( $valid_plan_ids, $subscriptions );
 	}
 
+	/**
+	 * The user is visiting with a subscriber token cookie
+	 * or the user is logged in and the blog subscription
+	 * is verified by POSTing the Auth endpoint
+	 */
+	public function vistor_is_blog_subscriber() {
```

Comment:
We need to remove that entire function, we should put that logic here:
https://github.com/Automattic/jetpack/blob/b313bd253afdf919adf7a71a866d0ab85b3a7bda/projects/plugins/jetpack/extensions/blocks/premium-content/_inc/subscription-service/class-token-subscription-service.php#L50

We could not be doing any calls here

Reply:
I reverted https://github.com/Automattic/jetpack/pull/26417/commits/b909fb0b895d32885a876f938636f5c45f995eae to be continued elsewhere